File: fixtures/multi_language/src/php/DataProcessor.php
<?php

class DataProcessor
{
    private array $items = [];

    public function addItem(string $item): void
    {
        $this->items[] = $item;
    }

    public function getItems(): array
    {
        return $this->items;
    }

    public function count(): int
    {
        return count($this->items);
    }
}

File: fixtures/multi_language/src/csharp/DataProcessor.cs
using System;
using System.Collections.Generic;

namespace MultiLanguage
{
    public class DataProcessor
    {
        private readonly List<string> _items = new List<string>();

        public void AddItem(string item)
        {
            _items.Add(item);
        }

        public IReadOnlyList<string> GetItems()
        {
            return _items.AsReadOnly();
        }

        public int Count()
        {
            return _items.Count;
        }
    }
}
